test/testvm.py

```diff
@@ -350,12 +350,19 @@ def download_dir(self, source, dest):
 
         self.message("Downloading", source)
         self.message(" ".join(cmd))
-        try:
-            subprocess.check_call([ "rm", "-rf", dest ])
-            subprocess.check_call(cmd)
-            subprocess.check_call([ "find", dest, "-type", "f", "-exec", "chmod", "0644", "{}", ";" ])
-        except:
-            self.message("Error while downloading journal")
+        def run_commands():
+            try:
+                subprocess.check_call([ "rm", "-rf", dest ])
+                subprocess.check_call(cmd)
+                subprocess.check_call([ "find", dest, "-type", "f", "-exec", "chmod", "0644", "{}", ";" ])
+            except:
+                self.message("Error while downloading journal")
+        if quiet:
+            with stdchannel_redirected(sys.stdout, os.devnull), stdchannel_redirected(sys.stderr, os.devnull):
```

Comment:
This only affects download - scp with the options listed above. No scripts are executed.